pkg/roachprod/install/cockroach.go

```diff
@@ -221,7 +222,10 @@ func (c *SyncedCluster) Start(ctx context.Context, l *logger.Logger, startOpts S
 	}, DefaultSSHRetryOpts); err != nil {
 		return err
 	}
-	if startOpts.ScheduleBackups {
+
+	// Only after a successful cluster initialization should we attempt to schedule backups.
+	if startOpts.ScheduleBackups && !startOpts.SkipInit {
+		c.Cluster.VMs.Len()
```

Comment:
What's this for?

Reply:
heh. nothing. a silly mistake. removing.